### ISSUE TITLE
Introduction of python 3x

### ISSUE THREAD

**Issue by FarhaKousar1601** [FarhaKousar1601/Python_EBook_Free#2]
**Introduction of python**

Introduction to Python:

1. Python is a high-level, interpreted, and versatile programming language known for its simplicity and readability.
2. Created by Guido van Rossum and first released in 1991, Python has gained immense popularity due to its ease of use and powerful capabilities.
3. Python's design philosophy emphasizes code readability, favoring a clean and straightforward syntax that allows programmers to express concepts in fewer lines of code compared to other languages.

Code Example - Printing "Hello, World!":

```python
print("Hello, World!")
```

4. Python is an open-source language, meaning its source code is freely available, and developers can contribute to its ongoing development and improvement.
5. The language supports various programming paradigms, including procedural, object-oriented, and functional programming, allowing developers to choose the style that best suits their needs.

Code Example - Defining a Function to Calculate the Area of a Circle:

```python
def calculate_circle_area(radius):
    pi = 3.14159
    area = pi * radius ** 2
    return area

radius_input = float(input("Enter the radius of the circle: "))
print("The area of the circle is:", calculate_circle_area(radius_input))
```

6. Python has a vast standard library with modules and packages that offer a wide range of functionalities, making it suitable for various applications such as web development, data analysis, machine learning, and automation.

Code Example - Using the `math` Module to Calculate the Square Root:

```python
import math

number = float(input("Enter a number: "))
square_root = math.sqrt(number)
print("The square root of", number, "is", square_root)
```

7. Python's community-driven development has resulted in a rich ecosystem of third-party libraries and frameworks, making it easy for developers to find solutions and tools for almost any task.

Code Example - Using the `requests` Library to Fetch Data from a URL:

```python
import requests

url = "https://jsonplaceholder.typicode.com/posts/1"
response = requests.get(url)
data = response.json()

print("Title:", data['title'])
print("Body:", data['body'])
```

8. Python is a cross-platform language, supporting various operating systems like Windows, macOS, and Linux, ensuring that code can be executed on different platforms without modification.

9. The official Python website (https://www.python.org) provides extensive documentation, tutorials, and resources for both beginners and experienced developers, helping users get started and improve their Python skills.

In summary, Python's combination of simplicity, flexibility, and community support has made it a favorite among programmers of all levels, enabling them to build robust and innovative applications for diverse domains.
